pkg/maps/lbmap/lbmap.go

```diff
@@ -548,12 +548,12 @@ func serviceKeynValue2FEnBE(svcKey ServiceKey, svcValue ServiceValue) (*loadbala
 
 // RevNat6Value2L3n4Addr converts the given RevNat6Value to a L3n4Addr.
```

Comment:
@eloycoto there was a change in 
https://github.com/cilium/cilium/commit/77d07735cf0b9a05a7156fc693a3a94866956709 to function `serviceKeynValue2FEnBE` which is not present here - is that intentional?

Reply:
Seems missing. It's only cosmetic right now but a future backport might require that frontend and backend have matching protocol values.